test/text_to_binary.extension_test.cpp

```diff
@@ -145,19 +145,19 @@ INSTANTIATE_TEST_CASE_P(
                  MakeInstruction(SpvOpSubgroupBallotKHR, {1, 2, 3})},
                 {"%2 = OpSubgroupFirstInvocationKHR %1 %3\n",
                  MakeInstruction(SpvOpSubgroupFirstInvocationKHR, {1, 2, 3})},
-                {"OpDecorate %1 BuiltIn SubgroupEqMaskKHR\n",
+                {"OpDecorate %1 BuiltIn SubgroupEqMask\n",
```

Comment:
In SPIR-V 1.3 these toke numbers are pulled into core, so the preferred name is used here for the round trip assembly/disassembly test.  That drops the KHR suffix.